Source/Scene/SDFSettings.js

```diff
@@ -0,0 +1,48 @@
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    'use strict';
+
+    /**
+     * Settings for the generation of signed distance field glyphs
+     *
+     * @exports SDFSettings
```

Comment:
Should this file be `@private`?

Reply:
Yeah it should, at least for now.  In theory you shouldn't have to mess with those at all as a regular user.  Pushed a change for this.